apps/dashboard/services.py

```diff
@@ -73,6 +76,18 @@ def get_filtered_queryset_base(
             sessions = sessions.filter(experiment_channel__platform__in=platform_names)
             messages = messages.filter(chat__experiment_session__experiment_channel__platform__in=platform_names)
 
+        if participant_ids:
+            participants = participants.filter(id__in=participant_ids)
+            sessions = sessions.filter(participant__id__in=participant_ids)
+            messages = messages.filter(chat__experiment_session__participant__id__in=participant_ids)
+
+        if tag_ids:
+            tags = [int(t) for t in tag_ids]
```

Comment:
I think once https://github.com/dimagi/open-chat-studio/pull/1870#pullrequestreview-3016765431 is resolved this may not be required at all (and if it is, it should be done in the `DashboardFilterForm.clean_tags` method.